src/flick/flick_auth/controllers/register_controller.py

```diff
@@ -39,15 +41,32 @@ def _create_profile(self, profile_data):
         profile.save()
         return profile
 
+    def _check_token(self, token, type="facebook"):
+        """
+        checks if the token is valid, right now only supports facebook token
+        """
+        if not token:
+            return False
+        # only validate if it is turned on from settings, helps with easier development
+        if settings.VALIDATE_SOCIAL_TOKEN:
+            URL = "https://graph.facebook.com/me"
+            PARAMS = {"access_token": token}
+            response = requests.get(url=URL, params=PARAMS)
+            data = response.json()
+            if data.get("error"):
```

Comment:
is there no success param in the data? instead of checking for `error`, can you not check for `if not data.get("success")`?

Reply:
you could also just `return data.get("success")` since data usually looks like `{"success": True}` or `{"success": False}` but if it doesn't have the `"success"` parameter, you can just `return not data.get("error")`

Reply:
It does not have a success field. the success response looks like {name: vivi, id:123456}. The failture response has {error: "message"}